lnwallet/channel.go

```diff
@@ -4385,9 +4401,14 @@ func (lc *LightningChannel) ReceiveNewCommitment(commitSig lnwire.Sig,
 	// As an optimization, we'll generate a series of jobs for the worker
 	// pool to verify each of the HTLc signatures presented. Once
 	// generated, we'll submit these jobs to the worker pool.
+	var leaseExpiry uint32
+	if lc.channelState.ChanType.HasLeaseExpiration() {
```

Comment:
Could cut down on these three repeated lines by having a function that just returns zero if there's no lease expiry, and reads this value otherwise. Or even, just use `ThawHeight` directly, since we assume that it's always zero if this isn't a leased channel?

Reply:
This is what I had initially, but then ended up adding this check to make things explicit, as the `ThawHeight` could've previously been set from a channel lease that doesn't have script enforcement.